internal/app/recordtester/recordtester_app.go

```diff
@@ -301,8 +301,12 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 		return 0, err
 	}
 
+	lapiNoAPIKey := api.NewAPIClient(api.ClientOptions{
+		Server:      rt.lapi.GetServer(),
+		AccessToken: "", // test playback info call without API key
```

Comment:
Do we not have an available API key to use here? We actually wanted to get rid of the API-key-less support, since it causes some billing issues for example. We have a log in there to eventually remove it, but the deprecation was paused. 

What's the idea here not to use an API key for this call?